course/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+namespace Course
+{
+    enum Genre
+    {
+        Other,
+        Detective,
+        Adventure,
+        Horror,
+    }
+
+    class Author
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public DateTime BirthDate { get; init; }
+        public DateTime DeathDate { get; set; }
```

Comment:
for future - it should be nullable, author can be still alive :)
if you don't know what is nullable - just ignore)